### PR TITLE
Create Function.Disable-OutlookSignatures.ps1

### PR DESCRIPTION
**Pull request by sethbrasile**

### CODE REVIEW

File: Function.Disable-OutlookSignatures.ps1 [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  # Loop through list of user folders, deleting signatures for each user
+  Get-ChildItem -Path "C:\Users" | Where-Object { $_.Name -ne "Public" } | Foreach-Object {
+    $userPath = $_.FullName
+    $path = "$userPath\AppData\Roaming\Microsoft\Signatures"
+
+    If (Test-Path -Path $path) {
+      Get-ChildItem -Path $path | Remove-Item -Recurse -Force
+      $outputLog += "Deleted signatures at user path: $userPath"
+    }
+  }
```
Comment (wmmatt) [inline Function.Disable-OutlookSignatures.ps1:41]: Make sure this part is tested really, really well. Confirmation you've done this w/ test files of `fake.txt` or similar would be great before we turn it on.

File: Function.Disable-OutlookSignatures.ps1 [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  If (!(Get-ItemProperty 'HKLM:\SOFTWARE\Classes\Outlook.Application' -ErrorAction SilentlyContinue)) {
+    $outputLog += "Outlook is not installed so no further action is necessary."
```
Comment (wmmatt) [inline Function.Disable-OutlookSignatures.ps1:44]: Interesting if installed check...if it works that's a nice little shortcut.

**Review by wmmatt — COMMENTED** [review]
Just confirm on the tested deleting files and all good here!

File: Function.Disable-OutlookSignatures.ps1 (hunk 2 above)
Comment (sethbrasile) [reply Function.Disable-OutlookSignatures.ps1:44]: yep that's what ARI had in their script and as far as I can tell it does appear to be true if outlook is installed

File: Function.Disable-OutlookSignatures.ps1 (hunk 1 above)
Comment (sethbrasile) [reply Function.Disable-OutlookSignatures.ps1:41]: tested with files/folders that outlook generates, a folder named test and a file named fake.txt

File: Function.Disable-OutlookSignatures.ps1 (hunk 2 above)
Comment (wmmatt) [reply Function.Disable-OutlookSignatures.ps1:44]: Nice.

**Review by wmmatt — APPROVED** [review]

**Pull request merged**